cmd/placement/main.go

```diff
@@ -28,6 +33,8 @@ func main() {
 	metricsExporter := metrics.NewExporter(metrics.DefaultMetricNamespace)
 	metricsExporter.Options().AttachCmdFlags(flag.StringVar, flag.BoolVar)
 
+	flag.StringVar(&certChainPath, "certchain", "/var/run/dapr/credentials", "Path to the credentials directory holding the cert chain")
```

Comment:
Can this be moved as const to crednetials pkg, its used at multiple places in codebase in operator, sentry etc.

Reply:
for this, I'd rather keep it here as flags are directly tied to the process, and I'd want to keep the option of changing them if needed..

Reply:
sure, please make it a const string here.

Reply:
done